randovania/games/super_metroid/logic_database/Brinstar.txt

```diff
@@ -672,15 +689,28 @@ Extra - asset_id: 92
   * Layers: default
   * Normal Door to Big Pink/Door to Green Hill Zone
   > Door to Morph Ball Room
-      Hi-Jump Boots or Space Jump or Wall Jumps (Beginner)
+      Any of the following:
+          Hi-Jump Boots or Space Jump or Wall Jumps (Beginner)
+          # IBJ
+          Vertical Infinite Bomb Jump (Intermediate) and Can Lay Bombs
+          # HBJ
+          Horizontal Infinite Bomb Jump (Advanced) and Can Lay Bombs
   > Door to Noob Bridge
       Trivial
   > Pickup (Missile Expansion)
       All of the following:
-          Morph Ball
           Any of the following:
+              # Get through the pipe
+              Morph Ball
```

Comment:
Trickless should probably have some damage reqs to cross the platforming section. idk maybe 50?

Reply:
I'm trying to figure out what connection this comment is for. Is it the platforming to get to the MB tunnel? You can kill the bugs with a Power Beam shot to farm health if need be. I don't know if energy is necessary here unless you have less than 10 health for some reason.

Reply:
![grafik](https://github.com/randovania/randovania/assets/38186597/15afa8ba-f943-49f7-a23f-8e46874f1078)